core/include/swatch/core/AbstractMetric.hpp

```diff
@@ -35,6 +36,19 @@ enum Status {
   kNonCritical,
   kDisabled,
 };
+
+const boost::unordered_map<std::string, Status> StringToStatus= {
```

Comment:
Does it need to? Made it static now:
http://stackoverflow.com/questions/4806510/const-variables-in-header-file-and-static-initialization-fiasco